[0.166s] Invoking command in '/home/zzy/ur_real/build/ur_bringup': CMAKE_PREFIX_PATH=/opt/ros/humble CONDA_PROMPT_MODIFIER=(base) /usr/bin/cmake --build /home/zzy/ur_real/build/ur_bringup -- -j24 -l24
[0.196s] [35m[1mConsolidate compiler generated dependencies of target ur_bringup[0m
[0.209s] [ 50%] [32mBuilding CXX object CMakeFiles/ur_bringup.dir/src/check_collision.cpp.o[0m
[4.879s] [100%] [32m[1mLinking CXX executable ur_bringup[0m
[5.150s] /usr/bin/ld: /home/zzy/anaconda3/lib/libssl.so.3: undefined reference to `X509_STORE_CTX_init_rpk@OPENSSL_3.2.0'
[5.151s] /usr/bin/ld: /home/zzy/anaconda3/lib/libssl.so.3: undefined reference to `BIO_recvmmsg@OPENSSL_3.2.0'
[5.151s] /usr/bin/ld: /home/zzy/anaconda3/lib/libssl.so.3: undefined reference to `OSSL_ERR_STATE_restore@OPENSSL_3.2.0'
[5.152s] /usr/bin/ld: /home/zzy/anaconda3/lib/libssl.so.3: undefined reference to `OPENSSL_LH_set_thunks@OPENSSL_3.3.0'
[5.152s] /usr/bin/ld: /home/zzy/anaconda3/lib/libssl.so.3: undefined reference to `BIO_sendmmsg@OPENSSL_3.2.0'
[5.153s] /usr/bin/ld: /home/zzy/anaconda3/lib/libssl.so.3: undefined reference to `BIO_err_is_non_fatal@OPENSSL_3.2.0'
[5.153s] /usr/bin/ld: /home/zzy/anaconda3/lib/libssl.so.3: undefined reference to `BIO_get_rpoll_descriptor@OPENSSL_3.2.0'
[5.153s] /usr/bin/ld: /home/zzy/anaconda3/lib/libssl.so.3: undefined reference to `OSSL_ERR_STATE_free@OPENSSL_3.2.0'
[5.154s] /usr/bin/ld: /home/zzy/anaconda3/lib/libssl.so.3: undefined reference to `OSSL_STACK_OF_X509_free@OPENSSL_3.2.0'
[5.155s] /usr/bin/ld: /home/zzy/anaconda3/lib/libssl.so.3: undefined reference to `OSSL_ERR_STATE_save_to_mark@OPENSSL_3.2.0'
[5.155s] /usr/bin/ld: /home/zzy/anaconda3/lib/libssl.so.3: undefined reference to `OSSL_ERR_STATE_save@OPENSSL_3.2.0'
[5.156s] /usr/bin/ld: /home/zzy/anaconda3/lib/libssl.so.3: undefined reference to `BIO_get_wpoll_descriptor@OPENSSL_3.2.0'
[5.156s] /usr/bin/ld: /home/zzy/anaconda3/lib/libssl.so.3: undefined reference to `BIO_s_dgram_mem@OPENSSL_3.2.0'
[5.157s] /usr/bin/ld: /home/zzy/anaconda3/lib/libssl.so.3: undefined reference to `OSSL_ERR_STATE_new@OPENSSL_3.2.0'
[5.157s] /usr/bin/ld: /home/zzy/anaconda3/lib/libssl.so.3: undefined reference to `ERR_count_to_mark@OPENSSL_3.2.0'
[5.164s] collect2: error: ld returned 1 exit status
[5.165s] gmake[2]: *** [CMakeFiles/ur_bringup.dir/build.make:355ï¼šur_bringup] é”™è¯¯ 1
[5.165s] gmake[1]: *** [CMakeFiles/Makefile2:137ï¼šCMakeFiles/ur_bringup.dir/all] é”™è¯¯ 2
[5.165s] gmake: *** [Makefile:146ï¼šall] é”™è¯¯ 2
[5.168s] Invoked command in '/home/zzy/ur_real/build/ur_bringup' returned '2': CMAKE_PREFIX_PATH=/opt/ros/humble CONDA_PROMPT_MODIFIER=(base) /usr/bin/cmake --build /home/zzy/ur_real/build/ur_bringup -- -j24 -l24
